x-pack/solutions/security/plugins/security_solution/public/cloud_security_posture/components/alerts/alerts_preview.tsx

```diff
@@ -145,7 +145,7 @@ export const AlertsPreview = ({
             <EuiFlexItem>
               <EuiSpacer />
               <DistributionBar
-                stats={alertStats.reverse()}
+                stats={alertStats}
```

Comment:
I was looking at how `alertStats` is generated from `severityMap`, it seems it can't guarantee a consistent sort order for `alertStats` as the Map set logic seems to be depending on the iteration of `alertsData`, unless I missed something I think we would need to also change that logic

Reply:
Nice catch! I will take a look.